### PR TITLE
Add missing test with actual hostname

### PR DESCRIPTION
**Pull request by dineshba**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Adding one missed test case for method `GetHostname` in `pkg/util/node/node.go`

### CODE REVIEW

**Comment by dineshba** [comment]
I signed it

File: pkg/util/node/node_test.go [hunk 1]
```diff
@@ -104,6 +108,11 @@ func TestGetHostname(t *testing.T) {
 			expectedHostName: "abc",
 			expectError:      false,
 		},
+		{
+			hostName:         "",
+			expectedHostName: strings.ToLower(hostname),
+			expectError:      false,
```
Comment (neolit123) [inline pkg/util/node/node_test.go]: this binds the test to the underlying implementation, using:
- os.Hostname
- ToLower()

leaving the LGTM / approve to /utils/node maintainers.

Comment (neolit123) [reply pkg/util/node/node_test.go]: if you are not getting reviews for this, try the #pr-reviews /  #sig-node channels on the k8s slack.

File: pkg/util/node/node_test.go [hunk 2]
```diff
@@ -17,6 +17,9 @@ limitations under the License.
 package node
 
 import (
+	"net"
```
Comment (mattjmcnaughton) [inline pkg/util/node/node_test.go]: I think this is an unnecessary import - I "believe" removing this should cause the test suite (or at least the verify test) to pass.

Comment (dineshba) [reply pkg/util/node/node_test.go]: Thanks @mattjmcnaughton! Updated it

**Pull request closed without merging**